collateral-onboarding/collateral-onboarding/onboard.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+cd ~/spells/spells-mainnet
+dapp update
+make
+dapp create DssSpell
```

Comment:
This `make all` only builds them, but doesn't execute
What `dapp create DssSpell` actually do? Can 1+ particular spell be executed instead?